common/controllers/form-wizard.js

```diff
@@ -17,14 +18,32 @@ class FormController extends Controller {
     next()
   }
 
+  setupAddMultipleFieldsValues(req, res, next) {
+    const values = req.sessionModel.toJSON()
+
+    const addAnotherProps = Object.keys(req.form.options.fields).filter(
+      key => req.form.options.fields[key].component === 'appAddAnother'
+    )
+
+    addAnotherProps.forEach(fieldName => {
+      const fieldValues =
+        req.body?.[fieldName] || values.errorValues?.[fieldName]
+
+      if (fieldValues) {
+        req.sessionModel.set(fieldName, fieldValues)
+      }
+    })
+
+    next()
+  }
+
   setupAddMultipleFields(req, res, next) {
     const allFields = req.form.options.allFields
     const values = req.sessionModel.toJSON()
     const itemFields = Object.entries(req.form.options.fields).reduce(
       fieldHelpers.reduceAddAnotherFields(allFields, values),
       {}
     )
-
```

Comment:
Any reason for this? Might be nice to keep the line break between blocks for readability.

Reply:
Reinstated